embedded_gems/pathogen/Gemfile

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gemspec
+
+gem 'rails', '~> 7.1.3.2'
```

Comment:
**nitpick (blocking):** Maybe just make this `>= 7, < 8` for now? Since this gem doesn't need to be hardcoded to a specific version as that would make it a pain when upgrading rail that we now need to update the gems dependency.